surefire-its/src/test/resources/surefire-1426-ignore-fail-jvm-crash/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.maven.plugins.surefire</groupId>
+  <artifactId>SUREFIRE-1426</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>SUREFIRE-1426</name>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${surefire.version}</version>
+        <configuration>
+          <argLine>-Dfile.encoding=UTF-8 -Duser.language=en -XFFOOOBEEER -Duser.region=US -showversion -Xmx6g -Xms2g -XX:+PrintGCDetails </argLine>
```

Comment:
There are multiple use cases for `maven.test.failure.ignore`.
This test with `-XFFOOOBEEER` already exists in `Surefire735ForkFailWithRedirectConsoleOutputIT`. Of course it must not be modified. You can only add a new test method and add the system property `maven.test.failure.ignore`.
And why `-Xmx6g -Xms2g`? No idea!

Reply:
> There are multiple use cases for `maven.test.failure.ignore`. This test with `-XFFOOOBEEER` already exists in `Surefire735ForkFailWithRedirectConsoleOutputIT`. Of course it must not be modified. You can only add a new test method and add the system property `maven.test.failure.ignore`. 

none with both `maven.test.failure.ignore` and jvm crash.
is there a real problem adding a new test and name it with the JIra ID?


> And why `-Xmx6g -Xms2g`? No idea!

why? the jvm doesn't even start because of a wrong parameter.
It could have been even `-Xmx128g -Xms64g` **THE JVM DO NOT START ON PURPOSE FOR THE TEST**